### PR TITLE
Add dotenv plugin to prod webpack config

### PR DESCRIPTION
**Pull request by peter7z**
Production webpack configuration as missing dotenv plugin.
Thanks @acollazomayer for finding this issue.

### CODE REVIEW

**Comment by bernabe9** [comment]
Was this change intentional: https://github.com/rootstrap/react-base/commit/b43e03eb66246b8143faa5ef6627dd81aa1d5ac1#diff-9968b87917a60f452c8ff987566a38e7 ? @agustito37

**Comment by agustito37** [comment]
Yes it was, I needed the `process.env.GZIP_ENABLED` variable from `.env` so I decided to not use the webpack plugin and load them directly.

> Was this change intentional: [b43e03e#diff-9968b87917a60f452c8ff987566a38e7](https://github.com/rootstrap/react-base/commit/b43e03eb66246b8143faa5ef6627dd81aa1d5ac1#diff-9968b87917a60f452c8ff987566a38e7) ? @agustito37

File: webpack/webpack.config.prod.js [hunk 1]
```diff
@@ -79,7 +78,9 @@ export default {
         test: /\.js$|\.css$/,
         threshold: 0,
         minRatio: 2
-      })
+      }),
+
+    new Dotenv({ path: envPath })
```
Comment (agustito37) [inline webpack/webpack.config.prod.js]:
```suggestion
    new Dotenv({ path: envPath, systemvars: true })
```

Comment (peter7z) [reply webpack/webpack.config.prod.js]: I would rather not expose system vars on a production build...

**Pull request merged**